AndroidManifest.xml

```diff
@@ -204,6 +209,7 @@
         </receiver>
 
         <service android:name=".services.observer.FileObserverService" />
+        <service android:name=".services.observer.InstantUploadFolderObserverService"/>
```

Comment:
I'd rather add functionality to ```FileObserverService```  instead of adding a new ```Service``` to the app. There are too many services right now.